tools/vlsvdiff.cpp

```diff
@@ -40,6 +40,7 @@
 
```

Comment:
So I am not now going through vlsvdiff as I hope/trust it works. If it is working, I suggest that field variables be included in the output variable and to-be-diffed lists of the testpackage, where relevant (i.e. not the trans and acc dedicated tests of course).